thundra/plugins/trace/trace_plugin.py

```diff
@@ -27,7 +27,7 @@ def before_invocation(self, data):
         context = data['context']
         event = data['event']
 
-        context_id = str(uuid.uuid4())
+        context_id = data['contextId']
         self.start_time = time.time() * 1000
```

Comment:
Can we use `datetime.datetime.now()` instead of `time.time()` for more accurate (milliseconds instead of seconds) time measurement?